### DISCUSSION TITLE
Coraza Caddy Plugin Install

### DISCUSSION THREAD

**Discussion by Talisker69** [corazawaf/coraza#1014] [General]
Hi,
What is the best way to install the coraza plugin for Caddy ? I allready installed the package of Caddy with dnf install because Fedora repos have the lastest version. Caddy works as reverse proxy. So now how to install coraza plugin ? do I need to use xcaddy ? Thanks in advance ....

**Comment by Talisker69** [comment]
Maybe I asking obsious questions but I very new in Caddy / Go and I like to understand instead of launching a ton of commands ...
Ok here's my understanding.
We have two main options to manage module or plugin for Caddy : 
1 : Use xCaddy a tools developped in Go to build/compile a version of Caddy with the Coraza module but before we need to install Go language (go-lang package) to build/compile xcaddy himself
2 : Install Caddy binaries with dnf (fedora 39) and Go-Lang to build/compile the Coraza module.
In my understanding, the version of Coraza module  is compatible with many versions of Caddy (that is not the case between Nginx and Modsecurity for instance. That mean, in a regular basis we can update caddy (dnf update) whitout problems.
So I will prefer the solution two to reduce the maintenance task.
Do i'm right ?

**Reply by jptosso** [reply]
@Venous-W second time, we do not accept messages in other languages than english. We understand that you are interested in contributing, but please to keep everything consistent, use english. This comment will be flagged for deletion

**Comment by jptosso** [comment]
@Talisker69 we keep 100% regression compatibility for Caddy v2.x, so as long as they do not launch v3 you should be totally safe to update
